Source/WebCore/page/LocalDOMWindow+CSSOM.idl

```diff
@@ -26,6 +26,6 @@
 typedef USVString CSSOMString;
 
 // https://drafts.csswg.org/cssom/#extensions-to-the-window-interface
-partial interface LocalDOMWindow {
+partial interface DOMWindow {
```

Comment:
Should this file be renamed now? Same with all the other `LocalDOMWindow+*.idl` files.

Reply:
That's the cleanup I mentioned.  I'll rename in a future PR.